supplement.csv

```diff
@@ -1,7 +1,93 @@
 Lemma,Sense,Word Class,Year,Quotation,Source,Note
+愛國者,/,noun,1899,有憂國者。有愛國者。+,梁啓超『自由書·憂國與愛國』, quote taken from 1934 edition of 『飲冰室全集』
```

Comment:
```suggestion
愛國者,/,noun,1899,有憂國者。有愛國者。,梁啓超『自由書·憂國與愛國』+, quote taken from 1934 edition of 『飲冰室全集』
```
Oh, I think I misunderstood you.

The way `+` is used so far actually includes both multiple attestations from the same source and from multiple sources. I’ll revise Conventions.